lib/jdomodels/src/main/resources/schema/JsonSchemaBindObject-ddl.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS `JSON_SCHEMA_OBJECT_BINDING` (
+  `BIND_ID` BIGINT NOT NULL,
+  `SCHEMA_ID` BIGINT NOT NULL,
+  `VERSION_ID` BIGINT DEFAULT NULL,
+  `OBJECT_ID` BIGINT NOT NULL,
+  `OBJECT_TYPE` ENUM ('entity'),
```

Comment:
You might want to consider to mirror the OBJECT_TYPE value as the internal OBJECT_TYPE (which is uppercase)